clang/test/Interpreter/crash.cpp

```diff
@@ -0,0 +1,20 @@
+// REQUIRES: host-supports-jit, x86_64-linux
+
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+//
+// RUN: split-file %s %t
+//
+// RUN: %clang++ -std=c++20 -fPIC -c %t/vec.cpp -o %t/vec.o
+// RUN: %clang++ -shared %t/vec.o -o %t/vec.so
```

Comment:
This invokes the system linker. From the build system's point of view, it's almost never correct to invoke the system linker without the environment variables `$LDFLAGS` and `$LIBS`. In this case, you expect to make the shared library to link against the default libs. 

I am thinking of changing this line to read 
```
// RUN: %clang++ -shared %t/vec.o -o %t/vec.so $LDFLAGS $LIBS
```
Not particularly satisfied with this approach though.

I also think that your `vec.o` and `vec.so` are both empty after my previous fix. This suggests that the two steps here may not even be needed at all for the sake of testing this fix.

Reply:
Perhaps we should use this yaml-to-so tool to build the library.

Reply:
> Perhaps we should use this yaml-to-so tool to build the library.

Can you point me to the tool? A naive search returned nothing.

Reply:
If there's no quick solution, I would propose to revert the testing portion of this patch. This has been blocking CI on our end for a couple of days. No other tests in llvm repo invokes system linker like this one.

Reply:
The tools is called `yaml2obj`.